templates/04-web.yaml

```diff
@@ -372,8 +372,8 @@ Resources:
   WebAutoScalingGroup:
     Type: AWS::AutoScaling::AutoScalingGroup
     Properties:
-      Cooldown: '60'
-      HealthCheckGracePeriod: 150
+      Cooldown: 60
+      HealthCheckGracePeriod: 3500
```

Comment:
@sharepointoscar Why such a big increase from 150 to 3500?

Reply:
Yep was testing, and set back to a lower value now.